webapp/sentry.client.config.js

```diff
@@ -0,0 +1,22 @@
+import * as Sentry from '@sentry/nextjs'
+
+Sentry.init({
+  dsn: process.env.NEXT_PUBLIC_SENTRY_DSN,
+  enabled: !!process.env.NEXT_PUBLIC_SENTRY_DSN,
+  environment: process.env.NEXT_PUBLIC_SENTRY_ENVIRONMENT || 'staging',
+  integrations: [
+    // This overrides the default implementation of the RewriteFrames
+    // integration from sentry/nextjs. It adds the decodeURI() to fix a mismatch
+    // of sourceMaps in reported issues.
+    // ref: https://github.com/getsentry/sentry/issues/19713#issuecomment-696614341
+    Sentry.rewriteFramesIntegration({
+      iteratee(frame) {
+        const { origin } = new URL(frame.filename)
+        frame.filename = decodeURI(frame.filename.replace(origin, 'app://'))
+        return frame
+      },
+    }),
+  ],
+  release: `portal@${process.env.NEXT_PUBLIC_RELEASE_VERSION}`,
+  tracesSampleRate: 1.0,
```

Comment:
Can we use an env var here (set in the repo and read at build time)?

Reply:
Done here [ef2b1de](https://github.com/hemilabs/ui-monorepo/pull/622/commits/ef2b1de67435110054c0cbaa0bd146c746414d4b)